2025-08-10 11:58:43,253 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 11:58:43,253 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 11:58:44,223 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 11:58:44,223 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:00:04,021 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:00:04,021 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:00:05,066 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:00:05,066 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:00:45,251 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:00:45,251 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:00:46,292 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:00:46,292 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:05:38,414 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:05:38,414 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:05:39,545 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:05:39,545 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:06:28,989 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:06:28,989 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:06:30,068 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:06:30,068 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:07:08,597 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:07:08,597 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:07:09,607 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:07:09,607 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:07:30,707 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:07:30,707 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:07:31,790 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:07:31,790 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:07:51,486 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:07:51,486 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:07:52,486 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 12:07:52,486 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 23:04:03,039 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 23:04:03,039 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 23:04:04,102 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-10 23:04:04,102 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-11 00:24:00,490 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-11 00:24:00,490 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-11 00:24:01,355 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-11 00:24:01,355 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-11 01:51:00,174 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-11 01:51:00,174 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-11 01:51:01,260 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-11 01:51:01,260 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-12 10:38:58,596 - LSTM_Model - INFO - Starting model training...
2025-08-12 10:39:16,199 - LSTM_Model - INFO - Starting model training...
2025-08-12 10:39:49,283 - LSTM_Model - INFO - Starting model training...
2025-08-12 10:40:03,128 - LSTM_Model - INFO - Starting model training...
2025-08-12 10:45:35,052 - LSTM_Model - INFO - Training accuracy: 0.5181
2025-08-12 10:45:35,052 - LSTM_Model - INFO - Test accuracy: 0.4980
2025-08-12 10:45:35,088 - LSTM_Model - INFO - Model saved to /workspace/models//lstm_model_quick_20250812_104535.h5
2025-08-12 20:53:04,433 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-12 20:53:51,099 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-12 21:27:01,492 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-12 21:27:01,492 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-12 21:27:02,218 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-12 21:27:02,218 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-14 18:00:08,321 - LSTM_Model - INFO - Starting model training...
2025-08-14 18:15:21,579 - LSTM_Model - INFO - Starting model training...
2025-08-14 18:15:29,231 - LSTM_Model - INFO - Starting model training...
2025-08-14 18:30:45,255 - LSTM_Model - INFO - Starting model training...
2025-08-14 22:04:50,554 - LSTM_Model - INFO - Starting model training...
2025-08-14 22:04:58,801 - LSTM_Model - ERROR - Training failed with error: Input X contains infinity or a value too large for dtype('float64').
2025-08-14 22:04:58,803 - LSTM_Model - ERROR - Traceback (most recent call last):
  File "/workspace/lstm_model.py", line 277, in train_model
    features_scaled = self.feature_scaler.fit_transform(features)
  File "/home/ubuntu/.local/lib/python3.13/site-packages/sklearn/utils/_set_output.py", line 316, in wrapped
    data_to_wrap = f(self, X, *args, **kwargs)
  File "/home/ubuntu/.local/lib/python3.13/site-packages/sklearn/base.py", line 894, in fit_transform
    return self.fit(X, **fit_params).transform(X)
           ~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/.local/lib/python3.13/site-packages/sklearn/preprocessing/_data.py", line 907, in fit
    return self.partial_fit(X, y, sample_weight)
           ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/.local/lib/python3.13/site-packages/sklearn/base.py", line 1365, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/home/ubuntu/.local/lib/python3.13/site-packages/sklearn/preprocessing/_data.py", line 943, in partial_fit
    X = validate_data(
        self,
    ...<4 lines>...
        reset=first_call,
    )
  File "/home/ubuntu/.local/lib/python3.13/site-packages/sklearn/utils/validation.py", line 2954, in validate_data
    out = check_array(X, input_name="X", **check_params)
  File "/home/ubuntu/.local/lib/python3.13/site-packages/sklearn/utils/validation.py", line 1105, in check_array
    _assert_all_finite(
    ~~~~~~~~~~~~~~~~~~^
        array,
        ^^^^^^
    ...<2 lines>...
        allow_nan=ensure_all_finite == "allow-nan",
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/home/ubuntu/.local/lib/python3.13/site-packages/sklearn/utils/validation.py", line 120, in _assert_all_finite
    _assert_all_finite_element_wise(
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        X,
        ^^
    ...<4 lines>...
        input_name=input_name,
        ^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/home/ubuntu/.local/lib/python3.13/site-packages/sklearn/utils/validation.py", line 169, in _assert_all_finite_element_wise
    raise ValueError(msg_err)
ValueError: Input X contains infinity or a value too large for dtype('float64').

2025-08-14 22:20:20,680 - LSTM_Model - INFO - Created sequences: (940, 60, 24), targets: (940,)
2025-08-14 22:20:20,685 - LSTM_Model - INFO - Starting model training...
2025-08-14 22:20:20,685 - LSTM_Model - INFO - Starting model training...
2025-08-14 22:20:20,861 - LSTM_Model - INFO - Created sequences: (1940, 60, 24), targets: (1940,)
2025-08-14 22:20:20,861 - LSTM_Model - INFO - Created sequences: (1940, 60, 24), targets: (1940,)
2025-08-14 22:20:20,861 - LSTM_Model - INFO - Training data shapes - X: (1940, 60, 24), y: (1940,)
2025-08-14 22:20:20,861 - LSTM_Model - INFO - Training data shapes - X: (1940, 60, 24), y: (1940,)
2025-08-14 22:20:21,021 - LSTM_Model - INFO - Starting training with 5 epochs
2025-08-14 22:20:21,021 - LSTM_Model - INFO - Starting training with 5 epochs
2025-08-14 22:20:33,152 - LSTM_Model - INFO - Training accuracy: 0.5193
2025-08-14 22:20:33,152 - LSTM_Model - INFO - Training accuracy: 0.5193
2025-08-14 22:20:33,152 - LSTM_Model - INFO - Test accuracy: 0.5206
2025-08-14 22:20:33,152 - LSTM_Model - INFO - Test accuracy: 0.5206
2025-08-14 22:20:33,181 - LSTM_Model - INFO - Model saved to /workspace/models/test_model.h5
2025-08-14 22:20:33,181 - LSTM_Model - INFO - Model saved to /workspace/models/test_model.h5
2025-08-14 22:20:33,246 - LSTM_Model - INFO - Model loaded from /workspace/models/test_model.h5
2025-08-14 22:20:33,246 - LSTM_Model - INFO - Model loaded from /workspace/models/test_model.h5
2025-08-14 22:20:33,246 - LSTM_Model - INFO - Model loaded from /workspace/models/test_model.h5
2025-08-14 22:21:51,504 - LSTM_Model - INFO - Starting model training...
2025-08-14 22:21:51,695 - LSTM_Model - INFO - Created sequences: (9940, 60, 24), targets: (9940,)
2025-08-14 22:21:51,695 - LSTM_Model - INFO - Training data shapes - X: (9940, 60, 24), y: (9940,)
2025-08-14 22:21:51,946 - LSTM_Model - INFO - Starting training with 10 epochs
2025-08-14 22:23:20,605 - LSTM_Model - INFO - Training accuracy: 0.5234
2025-08-14 22:23:20,606 - LSTM_Model - INFO - Test accuracy: 0.5231
2025-08-14 22:23:20,645 - LSTM_Model - INFO - Model saved to /workspace/models/production_lstm_20250814_222320.h5
2025-08-14 22:23:20,721 - LSTM_Model - INFO - Model loaded from /workspace/models/production_lstm_20250814_222320.h5
2025-08-14 22:23:20,721 - LSTM_Model - INFO - Model loaded from /workspace/models/production_lstm_20250814_222320.h5
2025-08-14 22:23:20,788 - LSTM_Model - INFO - Model loaded from /workspace/models/production_lstm_20250814_222320.h5
2025-08-14 22:23:20,788 - LSTM_Model - INFO - Model loaded from /workspace/models/production_lstm_20250814_222320.h5
2025-08-14 22:23:20,788 - LSTM_Model - INFO - Model loaded from /workspace/models/production_lstm_20250814_222320.h5
2025-08-16 09:31:54,221 - LSTM_Model - INFO - Starting model training...
2025-08-16 09:31:54,409 - LSTM_Model - INFO - Created sequences: (12322, 60, 24), targets: (12322,)
2025-08-16 09:31:54,432 - LSTM_Model - INFO - Training data shapes - X_train: (9845, 60, 24), X_val: (2477, 60, 24)
2025-08-16 09:31:54,573 - LSTM_Model - INFO - Starting training with 5 epochs
2025-08-16 09:32:48,605 - LSTM_Model - INFO - Training accuracy: 0.4196
2025-08-16 09:32:48,606 - LSTM_Model - INFO - Test accuracy: 0.3222
2025-08-16 09:32:49,863 - LSTM_Model - INFO - Calibrated temperature: 3.000
2025-08-16 09:32:49,900 - LSTM_Model - INFO - Model saved to /workspace/models/production/lstm_real_data_20250816_093249.h5
2025-08-16 13:19:52,767 - LSTM_Model - INFO - Starting model training...
2025-08-16 13:19:55,266 - LSTM_Model - INFO - Created sequences: (123908, 60, 24), targets: (123908,)
2025-08-16 13:19:55,567 - LSTM_Model - INFO - Training data shapes - X_train: (99114, 60, 24), X_val: (24794, 60, 24)
2025-08-16 13:19:55,703 - LSTM_Model - INFO - Starting training with 50 epochs
2025-08-16 13:35:37,549 - LSTM_Model - INFO - Starting model training...
2025-08-16 13:35:37,739 - LSTM_Model - INFO - Created sequences: (9940, 60, 24), targets: (9940,)
2025-08-16 13:35:37,760 - LSTM_Model - INFO - Training data shapes - X_train: (7940, 60, 24), X_val: (2000, 60, 24)
2025-08-16 13:35:37,910 - LSTM_Model - INFO - Starting training with 20 epochs
2025-08-16 13:38:18,470 - LSTM_Model - INFO - Training accuracy: 0.3455
2025-08-16 13:38:18,471 - LSTM_Model - INFO - Test accuracy: 0.3475
2025-08-16 13:38:19,641 - LSTM_Model - INFO - Calibrated temperature: 0.800
2025-08-16 13:38:19,686 - LSTM_Model - INFO - Model saved to /workspace/models/production_lstm_trained.h5
2025-08-16 13:38:19,734 - LSTM_Model - INFO - Created sequences: (1940, 60, 24), targets: (1940,)
2025-08-16 13:39:38,004 - LSTM_Model - INFO - Model loaded from /workspace/models/production_lstm_trained.h5
2025-08-16 13:39:39,129 - LSTM_Model - INFO - Model loaded from /workspace/models/production_lstm_trained.h5
2025-08-16 13:39:39,129 - LSTM_Model - INFO - Model loaded from /workspace/models/production_lstm_trained.h5
2025-08-16 13:59:03,631 - LSTM_Model - INFO - Model loaded from /workspace/models/production_lstm_trained.h5
2025-08-16 14:01:27,247 - LSTM_Model - INFO - Model loaded from /workspace/models/production_lstm_trained.h5
2025-08-28 18:47:04,004 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-28 18:47:04,004 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-28 18:47:04,727 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-28 18:47:04,727 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-28 18:47:47,197 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-28 18:48:06,518 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-28 18:48:08,300 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-28 18:48:08,300 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
2025-08-28 18:52:01,523 - LSTM_Model - ERROR - Failed to load model: [Errno 2] Unable to synchronously open file (unable to open file: name = '/workspace/models//lstm_trading_model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
